app/workers/articles/bust_multiple_caches_worker.rb

```diff
@@ -0,0 +1,14 @@
+module Articles
+  class BustMultipleCachesWorker
+    include Sidekiq::Worker
+    sidekiq_options queue: :default, retry: 10
```

Comment:
Can we move this to the `low_priority` queue? I know that the default queue has the same priority as the `low_priority` queue but I think putting it on the named queue makes it more clear what its priority is.

Reply:
@duduribeiro I think you're missing only this change and then we're set!